### PR TITLE
Test trailing commas

### PR DESCRIPTION
**Pull request by aljazerzen**

### CODE REVIEW

File: tests/sqlparser_common.rs [hunk 1]
```diff
@@ -6876,3 +6886,32 @@ fn parse_non_latin_identifiers() {
...
+#[test]
+fn parse_trailing_comma() {
+    let trailing_commas = TestedDialects {
+        dialects: vec![Box::new(GenericDialect {})],
+        options: Some(ParserOptions {
+            trailing_commas: true,
+        }),
+    };
+
+    // We shouldn't verify, because trailing commas will not get generated back
```
Comment (alamb) [inline tests/sqlparser_common.rs:6899]: How about using `one_statement_parses_to`: https://github.com/sqlparser-rs/sqlparser-rs/blob/3b1076c194d40f473b9eb3d1194d90202465d284/src/test_utils.rs#L79-L103

**Review by alamb — COMMENTED** [review]
Thanks @aljazerzen  -- this looks very nice

cc @ankrgyl

**Comment by aljazerzen** [comment]
Updated, ready to re-review.

**Review by alamb — APPROVED** [review]

**Comment by alamb** [comment]
Thanks again @aljazerzen

**Pull request merged**